Decred Wallet/Features/Settings/DeleteWalletConfirmationViewController.swift

```diff
@@ -23,6 +24,8 @@ class DeleteWalletConfirmationViewController: UIViewController, UITextFieldDeleg
         tap.cancelsTouchesInView = false
         self.view.addGestureRecognizer(tap)
         
+        self.deleteWalletHeader.text = "\(LocalizedStrings.deleteWalletDesc)?"
```

Comment:
What is `deleteWalletDesc`? Delete wallet description? Just checked and saw: `"deleteWalletDesc" = "Delete Wallet";` Why not `deleteWallet` as key?